core/src/main/python/wlsdeploy/aliases/alias_entries.py

```diff
@@ -87,6 +87,7 @@ class AliasEntries(object):
 
     __topology_top_level_folders = [
         'AdminConsole',
+        'CdiContainer',
```

Comment:
This will cover createDomain, but will need to be explicitly called out for update and deploy. If it's going to be part of deploy, maybe consider putting it under resources (all three do resources using the same code)

Any other opinions re: resources vs. topology?